kernel/component-manager/src/main/java/org/sakaiproject/component/api/ServerConfigurationService.java

```diff
@@ -213,6 +213,17 @@
 	 */
 	int getInt(String name, int dflt);
 
+	/**
+	 * Access some named configuration value as an double.
+	 *
+	 * @param name
+	 *        The configuration value name.
+	 * @param defaultValue
+	 *        The value to return if not found.
+	 * @return The configuration value with this name, or the default value if not found.
+	 */
+	double getDouble(String name, double defaultValue);
```

Comment:
Well, I'd like to avoid kernel changes if possible, but this makes sense for Sakai as a product.